js/newArticle.js

```diff
@@ -26,11 +48,30 @@ makeNewArticle.JSON = function() {
     makeNewArticle.publishedOn = new Date();
     makeNewArticle.body = marked($('#article-body').val());
 
-    genJSON = new Article(makeNewArticle);
+    var genJSON = new Article(makeNewArticle);
     $('#export-field').text(JSON.stringify(genJSON));
   });
 };
 
+
+// makeNewArticle.setToStore = function() {
+//   var setLocal = JSON.stringify(genJSON);
+//   localStorage.setItem("saved", setLocal);
+//   console.log('done');
+//   console.log(setLocal);
+// };
+// makeNewArticle.setToStore();
+
+// makeNewArticle.getFromStore = function (username) {
+//   var getLocal = localStorage.getItem(setLocal);
+//   var unstringedTemp;
+//   if(getLocal != null) {
+//     unstringedTemp = JSON.parse(getLocal);
+//   }
+// };
+// makeNewArticle.getFromStore("setLocal");
+
+
```

Comment:
Avoid committing commented out code to your master branch - it adds confusion and clutter. If you need to leave it in for some reason, state that reason along with the commented out block.